docs/_docs/04_reference/config-blocks-and-attributes.md

```diff
@@ -63,6 +63,18 @@ The `terraform` block supports the following arguments:
       registry]({{site.baseurl}}/docs/getting-started/quick-start#a-note-about-using-modules-from-the-registry) for more
       information about using modules from the Terraform Registry with Terragrunt.
 
+- `include_in_copy` (attribute): A list of glob patterns (e.g., `*.txt`) that should always be copied into the Terraform
```

Comment:
Something like this would be more clear:

```suggestion
- `include_in_copy` (attribute): A list of glob patterns (e.g., `["*.txt"]`) that should always be copied into the Terraform
```

Reply:
Done in https://github.com/gruntwork-io/terragrunt/pull/1941